src/main/java/org/folio/clients/SourceRecordStorageClient.java

```diff
@@ -58,4 +73,16 @@ public Optional<JsonObject> getRecordsByIds(List<String> ids, AbstractExportStra
     }
   }
 
+  private String getCentralTenantId(OkapiConnectionParams params) {
```

Comment:
As far as central tenant expected not to be often changed - is it possible to save this value in cache?